src/pages/dmt5-trading-signals/_signal-steps.tsx

```diff
@@ -239,52 +239,52 @@ const SignalSteps = ({ active_tab }: SignalStepsProps) => {
                                     label={content.subscriber.step_one_header}
                                     description={content.subscriber.step_one_text}
                                 >
-                                    <Show.Desktop min_width="992">
+                                    <Desktop breakpoint={992}>
```

Comment:
Could you pass in variable from `size` instead?

Reply:
tabletL has 992 , i will update the PR

Reply:
@sean-binary the default breakpoint value is tabletL, so I removed the breakpoint prop and updated the PR.